Translated en to de

JonnyJohnson opened the pull request.

jamorham commented:
Thank you for taking the time to do this. I will contact you as soon as I have had a chance to go through it and with any questions/queries.

jamorham commented:
I think typo should be `überschritten` instead of `übeschritten` ?
Also `Parakeet` instead of `Parakett` 
`Blutglukosewerte` instead of `Blutgukosewerte`
`splitten` instead of `spliten`
`geeignet` instead of `geeigent`
`Vorhersage` instead of `Verhersage`
`Ursprungszustand` instead of `Urpsrungszustand`

What do you think?

JonnyJohnson commented:
Hi,

thanks for your reply. As always, some typos were present, sorry about 
that. But all of your suggestions are correct.

Some other changes I would propose:

(line 50, letter removed)
Ersetzen Sie die Werte in {} mit Ihrer Werten
changed to
Ersetzen Sie die Werte in {} mit Ihren Werten

(line 51, changed port number to port)
mongodb://{Benutzer}:{Passwort}@{host}.mongolab.com:{port}/{Datenbank}
changed to
mongodb://{Benutzer}:{Passwort}@{host}.mongolab.com:{11111}/{Datenbank}

(line 126, changed to politer word ordering)
Bitte warten während der Sensor-Warmups. (
changed to
Bitte während der Sensor-Warmups warten. (

(line 143, changed to better word ordering)
Parakeet nicht halten
changed to
Parakeet unverändert lasen

(line 173. letter removed)
Vorhersage für Niedrige Werte
changed to
Vorhersage für Niedrig Werte

(line 186, correct order of words)
Einstellungen auf/von SDcard Laden/Speichern
changed to
Einstellungen von/auf SDcard Laden/Speichern

(line 193, letter removed)
Alarm für verpasste Werten changed to Alarm für verpasste Werte

(line 196, hyphen removed)
Persistenter Hoch-Alarm
changed to
Dauerhafter Hoch Alarm

(line 208, corrected spelling)
Nutze Trend Änderung
changed to
Nutze Trendänderung

(line 221, letter removed)
Einstellungen Datenquellene
changed to
Einstellung der Datenquelle

(line 240, comma removed)
Weitere wakelocks die eventuell für Nightwatch oder Android Wear 
benötigt werden (höherer Energieverbrauch)
changed to
Weitere wakelocks, die eventuell für Nightwatch oder Android Wear 
benötigt werden (höherer Energieverbrauch)

(line 251, letter removed)
Keine Alarme während Telefonaten
changed to
Kein Alarm bei Telefonaten

(line 264, letter removed)
Behandlungs-/Vorhersagekurves
changed to
Behandlungs-/Vorhersagekurve

(line 280, reordering of words)
Zukünftig Standard zu Sprach-Eingabe verwenden
changed to
Zukünftig automatisch Sprach-Eingabe verwenden

(line 305, corrected spelling)
Einstellungen auf externem Speicher sichern (sdcard).\n\nNach der 
Speicherung kann jede App auf (sensitive) Informationen 
zugreifen.\n\nDie Einstellungen sollten nach dem Import zugüg gelöscht 
werden.
changed to
Einstellungen auf externem Speicher sichern (SD-Karte).\n\nNach der 
Speicherung kann jede App auf (sensitive) Informationen 
zugreifen.\n\nDie Einstellungen sollten nach dem Import zügig gelöscht 
werden.

(line 313, shorter and better)
Sensor ist aufgewärmt, zum Start wird eine zweifache Kalibrierung benötigt.
changed to
Sensor ist bereit, zum Start zweifach kalibrieren.

(line 314, corrected spelling)
Um die Einrichtung abzuschließen, werdne zwei Blutglukosewerte benötigt.
changed to
Um die Einrichtung abzuschließen, werden zwei Blutglukosewerte benötigt.

(line 347, better word ordering)
Einstellungen in sdcard Downloads gespeichert
changed to
Einstellungen auf SD-Karte unter Downloads gespeichert

(line 349, capital letter)
Einstellungen geladen! - starte neu...
changed to
Einstellungen geladen! - Starte neu...

(line 354, shorter and better)
SD-Karte ist nicht beschreibbar - Speichern nicht möglich
changed to
SD-Karte nicht beschreibbar - Speichern nicht möglich

(line 355, shorter and better)
SD-Karte ist nicht lesbar
changed to
SD-Karte nicht lesbar

(line 361, always use Hoch)
Warnen, wenn Hoher Wert überschritten wird
changed to
Warnen, wenn Hoch Wert überschritten wird

(line 364, always use Hoch)
Permanent High Sound
changed to
Hoch Sound (durchgängig)

(line 366, better word ordering)
Warnungen bei Vorhersage anzeigen
changed to
Warnungen für Vorhersage anzeigen

(line 367, corrected spelling)
Benachrichtigen wenn vorhergesagt Zeitschwelle erreicht wird
changed to
Benachrichtigen, wenn Zeitschwelle in Vorhersage erreicht wird

(line 374, better words)
Warnungen starten lautlos und werden nach einer Zeitspanne aktiviert
changed to
Warnungen starten lautlos und werden nach einiger Zeit aktiviert

(line 374, corrected wor
Alarme Telefonaten lautlos
changed to
Alarme bei Telefonaten lautlos

(line 418, always use Werte)
Niedrige Zuckerwerte
changed to
Niedrige Werte

(line 422, don't translate dark)
Behandlung Farbe: dunkel
changed to
Behandlung Farbe: Dark

(line 426, removed hyphen)
8-Stunden-Durchschnitt als Linie
changed to
8 Stunden Durchschnitt als Linie

(line 527, removed hyphen)
24-Stunden-Durchschnitt als Linie
changed to
24 Stunden Durchschnitt als Linie

(line 484, changed comma to hyphen)
Nach Kalibrierung, Rohwerte an neuen Wert anpassen
changed to
Nach Kalibrierung: Rohwerte an neuen Wert anpassen

(line 486, removed multiple used word)
Zeitlinienrasterlinien sichtbar
changed to
Zeitrasterlinien sichtbar

(line 488, corrected word order)
Nützlich für Rausch- oder verpasste Werte
changed to
Nützlich bei Rauschen oder verpassten Werte

(line 493, changed words)
Vorhersagemodell inner workings
changed to
Vorhersagemodell (Erweiterte Anzeige)

(line 494, changed words)
Rauschmodell inner workings
changed to
Rauschmodell (Erweiterte Anzeige)


The corresponding .xml file is attached to this Email (if GitHub Email 
might remove the attachment, send me an Email so that I can make another 
pull request for this).

Regards


Am 22.07.2016 um 00:23 schrieb Jamorham:
>
> I think typo should be |überschritten| instead of |übeschritten| ?
> Also |Parakeet| instead of |Parakett|
> |Blutglukosewerte| instead of |Blutgukosewerte|
> |splitten| instead of |spliten|
> |geeignet| instead of |geeigent|
> |Vorhersage| instead of |Verhersage|
> |Ursprungszustand| instead of |Urpsrungszustand|
>
> What do you think?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/jamorham/xDrip-plus/pull/4#issuecomment-234402490>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGRH8QnWERX00YdXRUyxSnB_kcc9W1A9ks5qX_F7gaJpZM4JOxG4>.
>

jamorham commented:
The attachment didn't come through to github but you can just email it to me, my username at gmail.com - Please also tell me what name or alias you would like to be credited as in the list of translators? Thanks

JonnyJohnson commented:
Hi,

just as I expected ;) I wrote you a separate Email with the .xml file.
I've tested this with a custom build on a phone with a smaller display 
(4.3 inches). Some of the texts might be a few characters too long to 
show up. Nevertheless, on larger phones (5.5 inches) everything looks 
good. If possible, you can credit me as Jonny Johnson.

Another question I want to ask: Is there a possibility to change the 
static factors (Carb Absorption rate, glucose drop/Unit, etc.) to time 
dependent ones to improve the prediction models?

One last thing: Great work, keep it up! :)

Regards


Am 22.07.2016 um 10:09 schrieb Jamorham:
>
> The attachment didn't come through to github but you can just email it 
> to me, my username at gmail.com - Please also tell me what name or 
> alias you would like to be credited as in the list of translators? Thanks
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/jamorham/xDrip-plus/pull/4#issuecomment-234481065>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGRH8V3FvrucADAfPyzVLpYzTMGfYzftks5qYHqmgaJpZM4JOxG4>.
>

jamorham commented:
Okay my phone is 4.3 inches so I will double check that. 

To adjust the simulation parameters by time profile for carb ratio and insulin sensitivity you just click on them in the preferences to bring up the profile editor. Long press to split the first block and then adjust the time periods and values as you like. If you don't see the profile editor then upgrade to the latest nightly. It was added some time back though.

Carb absorption rate isn't included in this yet. I could add that if we think it would be useful. The linear model for that is quite simplistic compared to some better future model which might include factors like fat/protein content.

jamorham commented:
Merged
